media-plugins/mythplugins/mythplugins-30.0.ebuild

```diff
@@ -0,0 +1,168 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="xml"
+BACKPORTS="03f44039848bd09444ff4baa8dc158bd61454079"
+MY_P=${P%_p*}
+
+inherit python-single-r1 readme.gentoo-r1
+
+DESCRIPTION="Official WMythTV plugins"
+HOMEPAGE="https://www.mythtv.org"
+# mythtv and mythplugins are separate builds in the same github MythTV/mythtv repository
+SRC_URI="https://github.com/MythTV/mythtv/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-2+"
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+
+MYTHPLUGINS="+mytharchive +mythbrowser +mythgallery +mythgame \
++mythmusic +mythnetvision +mythnews +mythweather -mythzmserver +mythzoneminder"
```

Comment:
I'd urge you to re-think these `+` marks and **only** enable them on actually useful plugins. As you said, a couple of users will be using `mythmzmserver` so I bet there are other plugins that aren't as popular either. Anyway, that `-` does nothing and thus is useless.

Reply:
I selected the likely useful plugins only.